Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.8-slim
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt /app/
+
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . /app/
+
+RUN python manage.py migrate
```

Comment:
Running database migrations during the image build process is not recommended. It is better to run migrations as part of the container startup process to ensure the database is ready and accessible.